### PR TITLE
    [WIP] Add GitHub Actions config

### PR DESCRIPTION
**Pull request by openrichardfb**
Summary:
Per Omkar's advice, I am learning from the migration of [this](https://github.com/pytorch/audio/pull/3391/files#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47L244-L269) to [this](https://github.com/pytorch/audio/blob/main/.github/workflows/unittest-linux-cpu.yml)

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

Summary:

Documentation:

[Optional Fixes #issue]

Test Plan:

Please see a detailed explanation of how to fill out the fields in the relevant sections in PULL_REQUEST.md.

### CODE REVIEW

File: .github/workflows/glow-build.yml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+      fail-fast: false
+    uses: pytorch/test-infra/.github/workflows/linux_job.yml@main
+    with:
+      runner: linux.12xlarge
+      repository: pytorch/glow
+      timeout: 120
+      script: |
+        # echo '::group::Setup Environment Variables'
+        # Mark Build Directory Safe
+        # git config --global --add safe.directory /__w/audio/audio
```
Comment (osalpekar) [inline .github/workflows/glow-build.yml:26]: uncomment this line and change `audio` to `glow`

File: .github/workflows/glow-build.yml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+        # Set up Environment Variables
+        export PYTHON_VERSION="${{ matrix.python_version }}"
```
Comment (osalpekar) [inline .github/workflows/glow-build.yml:29]: can hardcode this to 3.8 and delete lines 14-17 from this file

**Review by osalpekar — APPROVED** [review]
LGTM, let's check this is in and see if it runs in GitHub Actions. The job should show up in the signalbox below but maybe this file needs to be added for GHA to be setup for the repo

File: .github/workflows/glow-build.yml (hunk 2 above)
Comment (openrichardfb) [reply .github/workflows/glow-build.yml:29]: Thanks Omkar!
I think glow stack does not care about 3.8 or 3.10. is remaining the old three version python gets it fits all or gets the CI runs 3 times?

I mean, ideally, we should be using whatever default python. So which way is better please?

Comment (osalpekar) [reply .github/workflows/glow-build.yml:29]: Looks like `build.sh` script is installing Python3.9 from source, so maybe we don't have to specify anything in the `glow-build.yml` file. I guess you can just remove any reference to python version (Line 14-17 and Line 29)

**Pull request closed without merging**